bundles/org.openhab.binding.smhi/src/main/resources/ESH-INF/config/forecast-config.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+		xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:smhi:forecast">
+		<parameter name="latitude" type="decimal" required="true">
+			<label>Latitude</label>
+			<description>Latitude for the forecast</description>
+		</parameter>
+		<parameter name="longitude" type="decimal" required="true">
+			<label>Longitude</label>
+			<description>Longitude for the forecast</description>
+		</parameter>
+		<parameter name="hourlyForecasts" type="integer" multiple="true">
+			<label>Hourly forecasts</label>
+			<description>The hourly forecasts to display</description>
+			<limitToOptions>true</limitToOptions>
+			<options>
+				<option value="0">00: Current hour</option>
```

Comment:
That's a workaround because paperUI sorts the option alphabetically, regardless of the order the options are specified. See https://github.com/openhab/openhab-webui/issues/212